### DISCUSSION TITLE
New level has no death plane.

### DISCUSSION THREAD

**Discussion by justagreenguy** [Techokami/SonicWorldsNext#165] [Q&A]
Hi! I made a new level, following the wiki (duplicating BaseZone, deleting everything below player), however now my level has no death plane on the bottom. I tried setting the bottom limit in the root node. Did I accidentally delete the node that works as the death plane, or is something else wrong?

**Comment by justagreenguy** [comment]
Nevermind, turns out it was because I was running the scene instead of the project.

**Reply by NeonSRB2** [reply]
Yeah, you can't die by any means.

**Reply by Caverns4** [reply]
I've been meaning to fix that so that only standard damage counts as "safe", just haven't gotten to it. I'll add it tomorrow.
